### ISSUE TITLE
Layout-break doesn't cause a column break in HA 2024.6

### ISSUE THREAD

**Issue by wheindl** [thomasloven/lovelace-layout-card#291]
My Home Assistant version: 2024.6

Layout-card version v2.4.5

What I am doing:  Just updated to 2024.6 (on two different HA instances).  The layout-break card no longer creates a column break; all cards appear in a single column.  The dashboard is set to Vertical (layout-card).

What I expected to happen:  The two or three columns in my dashboards would all continue to render as previously.

What happened instead:  All cards appear in a single column.

**Minimal** steps to reproduce:  Upgrade to HA 2024.6 and clear cache.

```yaml
# The least ammount of code possible to reproduce my error

title: Cameras
path: cameras
theme: ios-light-mode-blue-red-alternative
type: custom:vertical-layout
layout:
  width: 400
  max_width: 1000
  max_cols: 2
badges: []
cards:
  - type: vertical-stack
    cards:
  *   *    *
  - type: custom:layout-break

# End of code
```

Error messages from the browser console:

---

**By putting an X in the boxes ([X]) below, I indicate that I:**

- [x] Understand that this is a channel for reporting bugs, not a support forum (https://community.home-assistant.io/).
- [x] Have made sure I am using the latest version of the plugin.
- [x] Have followed the troubleshooting steps of the "Common Problems" section of https://github.com/thomasloven/hass-config/wiki/Lovelace-Plugins.
- [x] Understand that leaving one or more boxes unticked or failure to follow the template above may increase the time required to handle my bug-report, or cause it to be closed without further action.

**Comment by J-o-h-n-M** [comment]
Just spend some time understanding why my Google Nest Hub had a strange layout. And indeed it boiled down to the layout-break being ignored.

**Comment by snigehere** [comment]
I have the same issue - I didn't see anything in breaking changes for this

**Comment by terriblefox** [comment]
Same her since the lates update of HA. No layout break anymore.

**Comment by Leen15** [comment]
Same here

**Comment by Motik68** [comment]
Seems to still be the same bug as #288.

It is possible to restore the multi-column layout by editing the YAML of the tab as indicated there:

> changed:
> 
> ```
> type: custom:vertical-layout
> layout:
>   max_cols: 3
> ```
> to:
> 
> ```
> type: custom:layout-card
> layout_type: custom:vertical-layout
> layout:
>   max_cols: 3
> ```
> and it works again.

BUT... Then, when switching the tab to edit mode, the "Modify" buttons no longer appear and actually editing any card becomes impossible.

**Comment by 3shirts** [comment]
Same issue here. I love this because it's so simple to use so getting into the weeds to resolve it myself really isn't appealing!

**Comment by arboeh** [comment]
Same issue since upgrading core to 2024.6

**Comment by ferreto1978** [comment]
Same here. Updated to 2024.6 and no more breaks as a result. Reverted to 2024.5 which "fixed" it.

**Comment by mnneely** [comment]
I tried grid layouts and combinations of vertical stacks within horizontal stacks, but nothing reproduced the simplicity and clean function of the layout-break card. Like a *lot* of others, I eagerly await a fix to this incredibly useful and vital frontend.

**Comment by trisweb** [comment]
I also experience this issue after upgrading to 2024.6. What is the path to resolving it correctly? Is this project still maintained?

**Comment by larryb31** [comment]
Same..

**Comment by kloodhu** [comment]
same

**Comment by hellomarb** [comment]
Yes it's broken and setting `layout_type: custom:vertical-layout` isn't working for me on views.

**Comment by trisweb** [comment]
I went ahead and copy-pasted my dashboards to the new Drag and Drop dashboard layout, and it works quite well with similar Column concepts.

This spells the end of the road for Layout Card for me, and I expect due to the new layout tools, there will be decreasing need for it in the future. Maybe not time for everyone, but for me it works.

Thanks for all the fish. Cheers.

**Comment by 3shirts** [comment]
> I went ahead and copy-pasted my dashboards to the new Drag and Drop dashboard layout, and it works quite well with similar Column concepts.
> 
> This spells the end of the road for Layout Card for me, and I expect due to the new layout tools, there will be decreasing need for it in the future. Maybe not time for everyone, but for me it works.
> 
> Thanks for all the fish. Cheers.

Yep. I transferred one of my views to a "Sections" type today and it works great plus displays more consistently across devices for me. I think I'll probably be doing the rest of my views when I have time and then bin this.

**Comment by mnneely** [comment]
Ironically, I just did the same this morning. The Sections capability is good. I hope they make the ability to expand the width of certain cards (e.g. entity, button) to the whole section. I have several Mushroom Template cards that are assigned to two columns in a section row, which I would prefer to take up the whole row given the information they contain.

With layout-card:
<img width="521" alt="image" src="https://github.com/thomasloven/lovelace-layout-card/assets/8136142/dd2b24bd-a98c-4ffd-8003-315435f9c2d9">

With Sections:
<img width="475" alt="image" src="https://github.com/thomasloven/lovelace-layout-card/assets/8136142/981e8d0c-19bb-4a10-a5c1-3a5f09431175">

**Comment by mnneely** [comment]
Just found a work around. Add a vertical stack to the Section that contains the cards you want to occupy a whole row.

**Comment by 3shirts** [comment]
> Ironically, I just did the same this morning. The Sections capability is good. I hope they make the ability to expand the width of certain cards (e.g. entity, button) to the whole section. I have several Mushroom Template cards that are assigned to two columns in a section row, which I would prefer to take up the whole row given the information they contain.

Change the layout type on the card from "default" to either horizontal or vertical, then it will fill the whole row. I had the same issue at first.

**Comment by snigehere** [comment]
The worry with the section approach is the experimental nature of the feature.. if I switch all my dashboards over I may get issues in a later release if they have to change the architecture ....

**Comment by piitaya** [comment]
You can customize the width and the height on the card using `yaml` in section card. We are looking to bring that to the UI.
This feature is not documented yet because it's experimental and may change (although I don't think that will change 😅)

```yaml
layout_options:
   grid_rows: 4         # number of row for your card in the grid
   grid_columns: 4      # number of column for your card in the grid
```

**Comment by piitaya** [comment]
With 2024.6 card are now wrapped with a `hui-card` so the layout card needs to support that.

**Comment by arboeh** [comment]
I guess the issue still exists with core 2024.6.1 which has bern released 12h ago..?

**Comment by Leen15** [comment]
I moved to a grid-layout, it works as expected without using layout-break card.

**Comment by J-o-h-n-M** [comment]
I moved to a sections layout, was planning on this anyway and I am really happy I did. So much more flexibility

**Comment by snigehere** [comment]
much typing later .... 20 dashboards on two systems converted from using custom layout (only ever used the break capability of the card) to sections .. I now have to hope they don't change the section approach in a later release as it becomes final rather than experimental...  

A couple of issues with cards that have coding to not show if there is nothing to show - alexa timers and waste collection - no solution to these yet so a white space gap in the UI when they have nothing to display

**Comment by trisweb** [comment]
For those migrating to a Sections layout, you can actually adjust the yaml to convert your existing layout-card dashboard to a sections one, despite it warning you that migration is not supported.

First, I recommend backing up your dashboards entirely before you begin (hit edit, then click the ••• and pick "Raw configuration editor," then copy all into a backup file).

To do this:

1. Change `type: custom:vertical-layout` to `type: sections`
2. Remove `badges: []`
3. Add `max-columns: 3` (or your preference)
4. Above the 'cards' key, add a couple new lines:
   ```
   sections:
      - type: grid
   ```
5. Indent the existing 'cards' and everthing below it, to line up with the t in 'type', so it looks like, for ex:
   ```
   sections:
     - type: grid
        cards:
          - type: entities
            entities:
              - entity: light.living_room_lights
              - entity: light.dining_room_light
              ...
   ```
6. Now, you can either save it, making all of your cards flow into one column (where you can then drag them around in the UI)... or, you can add the other columns manually, wherever you had a `- type: custom:layout-break` line.
   
   In place of the `- type: custom:layout-break` line, add the following two lines to make a new column, reducing the indentation by two levels to make the things under 'cards:' line up properly.
   ```
   - type: grid
        cards:
   ```
   And the remaining cards go underneath the `cards:` key as with the first column.
   
   If you don't want to do this part, you can skip this step and just use the UI to make new columns and drag cards into them.

This made the migration much faster for me, but of course using copy-paste isn't so bad either if you can't do this.

**Comment by snigehere** [comment]
Many thanks for sharing ... i did similar on a couple of cards but for most of mine I found it easier to open each card in yaml one by one on the old dashboard and then add the card as manual in the section and paste the code in.. i found this quicker on more complex cards where there were lots of code lines and multiple views on the same dashboard .. adding 4 spaces on each row with copy paste got a bit tedious!!

**Comment by 500Foods** [comment]
That's good to know. The main thing holding me back from Sections is that I've not been able to figure out how to increase the width of a section (or all sections, doesn't have to be per-section, though that would be nice as well). Maybe I'm overlooking something obvious?

**Comment by larryb31** [comment]
I copied/pasted everything into a Sections dashboard. I set up my 3 columns as before, but I'm missing the width element that I used for a 7' tablet, I had:
```
width: 320
max_cols: 3
card_margin: 1px 7px 6px
```
Now I'm getting only 2 columns with the 3rd appearing under the first. 
Is there a way to use percentages or something for this new Sections view?

**Comment by piitaya** [comment]
I open a PR to fix the issue: https://github.com/thomasloven/lovelace-layout-card/pull/293

> That's good to know. The main thing holding me back from Sections is that I've not been able to figure out how to increase the width of a section (or all sections, doesn't have to be per-section, though that would be nice as well). Maybe I'm overlooking something obvious?
> Just that they're not as wide as they need to be 😦 And if someone knows how to address that, maybe they also know how to adjust the gap between sections? That'd be handy too.

There are several theme variables available for section view. That should fix your theme issues 🙂

- Max width of a section: `ha-view-sections-column-max-width` (default: `500px`)
- Min width of a section: `ha-view-sections-column-min-width` (default: `320px`)
- Horizontal gap between sections: `ha-view-sections-column-gap` (default: `32px`)
- Vertical gap between sections: `ha-view-sections-row-gap` (default: `8px`)
- Horizontal gap between cards: `ha-section-grid-column-gap` (default: `8px`)
- Vertical gap between cards: `ha-section-grid-row-gap` (default: `8px`)

**Comment by J-o-h-n-M** [comment]
> 
> Now I'm getting only 2 columns with the 3rd appearing under the first. On my desktop I see 3 columns. Is there a way to use percentages or something for this new Sections view?

I had the same, I was only seeing 2 columns, the reason is that the space between the columns when using Sections is too big. I reduced it by adding this line to my themes yaml file:
`ha-view-sections-column-gap: 10px`

**Comment by hellomarb** [comment]
@piitaya Fantastic, I tested the PR and the layout-break is working again, thank you!

**Comment by larryb31** [comment]
Where do I add the ha-view-sections-.. variables in the yaml - to the dashboard raw config?

**Comment by piitaya** [comment]
> Where do I add the ha-view-sections-.. variables in the yaml - to the dashboard raw config? what if I'm not using a theme?

You have to create a theme to add these variables. You can create one with these variables only.

**Comment by 500Foods** [comment]
> There are several theme variables available for section view. That should fix your theme issues 🙂
> 
>     * Max width of a section: `ha-view-sections-column-max-width` (default: `500px`)
> 
>     * Min width of a section: `ha-view-sections-column-min-width` (default: `320px`)
> 
>     * Horizontal gap between sections: `ha-view-sections-column-gap` (default: `32px`)
> 
>     * Vertical gap between sections: `ha-view-sections-row-gap` (default: `8px`)
> 
>     * Horizontal gap between cards within a section: `ha-section-grid-column-gap` (default: `8px`)
> 
>     * Vertical gap between cards within a section: `ha-section-grid-row-gap` (default: `8px`)

Absolutely fantastic! Thanks very much. I created a new theme by copying the contents of a Mushroom theme to a new file. And then just plunked in the values I wanted to change at the bottom.

```
Pintura Wide:
    # Home Assistant override
    ha-card-border-radius: 6px
    # Layout
    mush-spacing: 8px
    # Elements
    mush-icon-border-radius: 6px
    mush-control-border-radius: 6px
    # Chips
    mush-chip-border-radius: 6px
    modes:
        light: {}
        dark: {}
    mush-chip-spacing: 4px
    mush-chip-padding: 0 0.25em
    mush-chip-height: 32px
    mush-chip-font-size: 0.4em
    mush-chip-font-weight: normal
    mush-chip-icon-size: 0.7em
    mush-chip-avatar-padding: 0.1em
    mush-chip-avatar-border-radius: 50%
    mush-chip-background: var(--ha-card-background)
    #Sections
    ha-view-sections-column-max-width: 462px
    ha-view-sections-column-min-width: 462px
    ha-view-sections-column-gap: 8px
```
After an "All YAML Configuration" with the dev tools, and a Ctrl+Shift+R to force the reload (Firefox) things are looking pretty great. And FAR easier to move things around with drag and drop. I like it. A lot.

**Comment by pcb1962** [comment]
> I open a PR to fix the issue: #293

Hi Paul, I've tried and failed to build a new layout-card.js from your repo (Error: Your current platform "linux" and architecture "ia32" combination is not yet supported by the native Rollup build.) 
Any chance you could please upload a new version of layout-card.js with your changes as it doesn't look like Thomas is going to fix this anytime soon.

**Comment by hellomarb** [comment]
Here a build (on macOS) of layout-card.js (gzipped for github):
[layout-card.js.gz](https://github.com/user-attachments/files/15760068/layout-card.js.gz)

**Comment by andbad** [comment]
> Here a build (on macOS) of layout-card.js (gzipped for github): [layout-card.js.gz](https://github.com/user-attachments/files/15760068/layout-card.js.gz)

It works. Thank to you and @piitaya 

By(t)e

**Comment by pcb1962** [comment]
> Here a build (on macOS) of layout-card.js (gzipped for github):

Thank you, works perfectly.
I will migrate to grid or something eventually but I currently have around 40 dashboards using vertical layout so may take some time.

**Comment by snigehere** [comment]
hmm .. I am clearly doing something wrong, or I misunderstand what's needed, or I am just poking around in teh darkness with no idea what I am doing .. any are possible.  

I am using the the [custom vertical layout](custom:vertical-layout) card.  

Initially I just switched out the .js and restarted HA but that didn't solve the break issue for me.  
I then removed Thomas's version from my system, used Paul's fork url and installed as a custom repository ... code is all updated as of todays data so its installed ok however still break is not being handled

**Comment by pcb1962** [comment]
Have you cleared the browser cache? Most HA interface changes seem to require that.

**Comment by andbad** [comment]
> hmm .. I am clearly doing something wrong, or I misunderstand what's needed, or I am just poking around in teh darkness with no idea what I am doing .. any are possible.
> 
> I am using the the custom vertical layout card.
> 
> Initially I just switched out the .js and restarted HA but that didn't solve the break issue for me. I then removed Thomas's version from my system, used Paul's fork url and installed as a custom repository ... code is all updated as of todays data so its installed ok however still break is not being handled

In "Resources" add "?v=1" after "layout-card.js"
Something like:
/local/layout-card.js?v=1

Then reload frontend clearing the cache (you can do this with SHIFT+F5 key, or clicking on refresh page icon on the browser while press SHIFT key). Or clear all web browser cache.

By(t)e

**Comment by snigehere** [comment]
> Have you cleared the browser cache? Most HA interface changes seem to require that.

Thanks - I had not but now have .. restarted the browser after clearing as well but it didn't fix it .. tried Edge which I rarely use and I see the same issue 

So I started a fresh dashboard and set a view and set view type as vertical (layout card) .. I added a couple of cards and a break but it still all showed in one column.... 

```
views:
  - title: Home
    cards:
      - show_name: true
        show_icon: true
        type: button
        tap_action:
          action: toggle
        entity: light.garden_lights
      - type: custom:layout-break
      - type: gauge
        entity: sensor.red_freezer_energy_hourly
    type: custom:vertical-layout
```
I think I will go and lay down for a while...

**Comment by snigehere** [comment]
Thanks @andbad ... sorry to be thick - I am not familiar with url formats - I already have a hacs tag after the .js - is that just another with a ? or is there a separator I need to use?
eg 
/hacsfiles/lovelace-layout-card/layout-card.js?v=1?hacstag=62215851516834
or a separator like an & between the hasctag and v=1?
/hacsfiles/lovelace-layout-card/layout-card.js?v=1&hacstag=62215851516834

**Comment by andbad** [comment]
> Thanks @andbad ... sorry to be thick - I am not familiar with url formats - I already have a hacs tag after the .js - is that just another with a ? or is there a separator I need to use? eg /hacsfiles/lovelace-layout-card/layout-card.js?v=1?hacstag=62215851516834 or a separator like an & between the hasctag and v=1? /hacsfiles/lovelace-layout-card/layout-card.js?v=1&hacstag=62215851516834

I don't know how works hacstag, but I think you should use "&" as separator, like your second example.

By(t)e

**Comment by piitaya** [comment]
My PR doesn't include the built file because it must not be included in PR. So, adding my fork into hacs will not work.

You have to use this file https://github.com/thomasloven/lovelace-layout-card/issues/291#issuecomment-2157855768

**Comment by snigehere** [comment]
@piitaya Thanks Paul - so the answer "I am just poking around in the darkness with no idea what I am doing" was actually the correct choice.  The first thing I tried was to just replace the file however that didn't do it so I tried other steps .. what I neglected, and @pcb1962 reminded me, was to clear cache after replacing the file .... but by then I had already moved on to try other things so missed the eureka moment   ... Based on your post I went back to just replacing the file, then cleared cache, and all was back to working,  Many  thanks for your work on this very useful card.

I did try out sections but they didn't work for my family dashboard landing page as I have quite a few conditional cards designed to hide stuff not in use so as to keep the dashboard as compact as possible .. hiding sections didn't work as it maintained the grid and conditional cards were not playing well on my android 5.01 tablet fixed to the kitchen wall.

**Comment by larryb31** [comment]
All working with the ha-section-grid-row-... variables, thanks all!

**Comment by Montreal666** [comment]
> There are several theme variables available for section view. That should fix your theme issues 🙂
> 
> * Max width of a section: `ha-view-sections-column-max-width` (default: `500px`)
> * Min width of a section: `ha-view-sections-column-min-width` (default: `320px`)
> * Horizontal gap between sections: `ha-view-sections-column-gap` (default: `32px`)
> * Vertical gap between sections: `ha-view-sections-row-gap` (default: `8px`)
> * Horizontal gap between cards within a section: `ha-section-grid-column-gap` (default: `8px`)
> * Vertical gap between cards within a section: `ha-section-grid-row-gap` (default: `8px`)

Should this work under 2024.5.5 or only 2024.6.x  ?

I added this to my theme but doesn't seem to work despite empty cache/hard reload 
( still get the large gaps between cards within a section )

`
my-custom-theme:

  ha-section-grid-column-gap: 2px;
  ha-section-grid-row-gap: 2px;
`

**Comment by iptvcld** [comment]
I am trying to convert my layout to Sections as well but I am seeing large gaps going horizonal and vertically - is there a way i can adjust these?
![image](https://github.com/thomasloven/lovelace-layout-card/assets/34818686/b60c3f0b-5b48-4db9-9d88-b1979e121322)

**Comment by iptvcld** [comment]
> > I went ahead and copy-pasted my dashboards to the new Drag and Drop dashboard layout, and it works quite well with similar Column concepts.
> > This spells the end of the road for Layout Card for me, and I expect due to the new layout tools, there will be decreasing need for it in the future. Maybe not time for everyone, but for me it works.
> > Thanks for all the fish. Cheers.
> 
> Yep. I transferred one of my views to a "Sections" type today and it works great plus displays more consistently across devices for me. I think I'll probably be doing the rest of my views when I have time and then bin this.

Sections is still listed as experimental, maybe don’t invest all the time into that view yet. Does look really nice and easy to manage though.

**Comment by igorsantos07** [comment]
I mentioned the `--row-height: 66px` issue in [this comment](https://github.com/thomasloven/lovelace-auto-entities/issues/433#issuecomment-2161947274), and it also [affects Mushroom chips](https://github.com/piitaya/lovelace-mushroom/issues/1390).

@Montreal666, so far it doesn't seem there's a way to configure that value.

@iptvcld, you can create a custom theme and use in the dashboard, using [these variables posted earlier](https://github.com/thomasloven/lovelace-layout-card/issues/291#issuecomment-2156524817). This will solve all issues, except the vertical spacing you see below short cards.

As a summary, given I also had some hiccups trying to create my own theme as well:
1. create a yaml file under themes/whatever-you-name-it.yaml, like this (`Thin sections` is the theme name in this example:
```yaml
# origin: https://github.com/thomasloven/lovelace-layout-card/issues/291#issuecomment-2156524817
Thin sections:
    ha-view-sections-column-max-width:  500px   #default: 500
    ha-view-sections-column-min-width:  320px   #default: 320, tailored to half of a ultrawide screen with 5px of column gap
    ha-view-sections-column-gap:        32px    #default: 32
    ha-view-sections-row-gap:           8px     #default: 8
    ha-section-grid-column-gap:         8px     #default: 8
    ha-section-grid-row-gap:            8px     #default: 8
```
2. save that
3. make sure you have something similar to this in your `configuration.yaml`:
```yaml
frontend:
  themes: !include_dir_merge_named themes
```
4. reboot your instance if you had to change `configuration.yaml`
5. if you didn't reboot, or if you made further changes to the theme, you'll need to go to Developer Tools > Services and call `frontend.reload_themes`. That would refresh all themes, without the need for you to press F5 on the pages. Don't forget to select the theme in the Dashboard, Tab or Card.

Good luck!

**Comment by boehser-enkel** [comment]
So i edited the yaml to:
```
type: custom:layout-card
layout_type: custom:vertical-layout
```

replaced the layout-card.js and cleared the cache. 
The view is now okay but the edit buttons for the dashboard elements are still missing.
Did i miss something?

**Comment by snigehere** [comment]
Adding the layout type was a work around but caused the edit button issue.  The fix in the .js file made it all work again so you don't need layout type in the yaml

**Comment by iptvcld** [comment]
> vertical spacing you see below short card

Thank you; this worked and yes as for the vertical spacing for short cards - that still remains, any cool header ideas you have?  Also is this the time to jump ship and and use sections? Or is it too early?

**Comment by igorsantos07** [comment]
@iptvcld no cool ideas, unfortunately haha
I've so far moved to Sections as those issues seem minor to me and should be fixed in the near future. It feels better than using custom code from developers (even from someone as dedicated as Thomas, it's prone to break since it's not tested together with the HA releases); and drag-and-drop is a bliss as well.

**Comment by iptvcld** [comment]
> @iptvcld no cool ideas, unfortunately haha
> 
> I've so far moved my main dashboard to Sections, with tweaks to the custom theme I posted, converting with trisweb's guide ([#291 (comment)](https://github.com/thomasloven/lovelace-layout-card/issues/291#issuecomment-2156220648)). Those issues seem minor to me and should be fixed in the near future. It feels better than using custom code from developers (even from someone as dedicated as Thomas, it's prone to break since it's not tested together with the HA releases); and drag-and-drop is a bliss as well.

Yeah the drag and drop is a cool feature and also being able to add columns by dragging a slider is neat.  Looks like I will start copy and pasting my cards over to new Sections views as it cannot be migrated.

**Comment by tharward-r7** [comment]
@iptvcld instructions for manually migrating to Sections from Layout Card dashboards here if useful: https://github.com/thomasloven/lovelace-layout-card/issues/291#issuecomment-2156220648

**Comment by 500Foods** [comment]
> Yeah the drag and drop is a cool feature and also being able to add columns by dragging a slider is neat. Looks like I will start copy and pasting my cards over to new Sections views as it cannot be migrated.

If you look back through this thread, there are steps to migrate manually. I did this with a dashboard with dozens of complex cards and it only took about five minutes. Super happy with the result. Note that for the bit about having to indent everything (thousands of lines in my case) you can just highlight the text and hit tab (twice here) to make quick work of it.

**Comment by Landshark77** [comment]
is this mod even still being maintained??

**Comment by igorsantos07** [comment]
I guess Thomas took a break from GitHub life... The PR from Piitaya #293 is up for grabs for two weeks already.

**Comment by nathan951** [comment]
So I moved over to 'sections' because of this issue. But now I have another small problem. I'm getting a 2 column view on smaller widths (tablets), BUT I really want 3 columns. How do I do that?


From this:
![image](https://github.com/thomasloven/lovelace-layout-card/assets/24908162/6679bb9a-435d-4a30-8856-2ffd4a5d7295)

To this:
![image](https://github.com/thomasloven/lovelace-layout-card/assets/24908162/1a80365f-1889-41ae-b7ec-7ad4371f0362)


Here's what my header of this section looks like:

`type: sections
max-columns: 3
background: var(--background-image)
title: Home
wallpanel:
  fullscreen: true
  enabled: true
  hide_toolbar: true
  hide_sidebar: true
  idle_time: 0
  black_screen_after_time: 300
views:
  - type: sections
    max_columns: 3
    subview: false
    sections:
      - type: grid
        layout: null
        width: 200
        cards:
          - graph: line
            type: sensor
            entity: sensor.pool_temperature
            detail: 2
            hours_to_show: 48
          - graph: line
            type: sensor
            entity: sensor.openweathermap_forecast_temperature
            detail: 2
            hours_to_show: 48
            name: Ouside Temp
          - type: horizontal-stack`

**Comment by skank01** [comment]
Wow lots of comments here, whats the proper way now?

**Comment by pcb1962** [comment]
The 'proper' way is for Thomas to accept the PR and produce a new release.
Easiest thing to do right now is is to replace your layout-card.js with one of those created by @hellomarb or @sdrapha.
Otherwise it's up to you to use one of the migration strategies posted above to eliminate your dependency on layout-card.

**Comment by skank01** [comment]
Thank you, works again !

**Comment by raytedjaja** [comment]
still waiting on 2024.5 release

**Comment by N3orun** [comment]
> The 'proper' way is for Thomas to accept the PR and produce a new release. Easiest thing to do right now is is to replace your layout-card.js with one of those created by @hellomarb or @sdrapha. Otherwise it's up to you to use one of the migration strategies posted above to eliminate your dependency on layout-card.

Could you specify what i have to do?  - sorry noobie here.

**Comment by Nyxtorm** [comment]
Thank you, it works again with https://github.com/sdrapha/lovelace-layout-card/blob/fix_break_card_ha_2024.6/layout-card.js. 👍
Don't forget to replace file in `layout-card.js.gz` as well (or maybe just remove this gz file), and increase version in resource URL.

**Comment by raytedjaja** [comment]
> Thank you, it works again with https://github.com/sdrapha/lovelace-layout-card/blob/fix_break_card_ha_2024.6/layout-card.js. 👍 Don't forget to replace file in `layout-card.js.gz` as well (or maybe just remove this gz file), and increase version in resource URL.

can you provide the step please,

**Comment by craigmcfly** [comment]
Any news on this? I've had to remove the layout-card from my dashboard and just let HA decide where things go, because any layout-card I create just gets put under the existing cards and I have to scroll down on my screen to see everything.

**Comment by hellomarb** [comment]
Use one of the working layout-card.js files in this post or from the pr (https://github.com/thomasloven/lovelace-layout-card/pull/293) and it is working again.
